docs/backups-scheduled.md

```diff
@@ -1,40 +1,112 @@
+    * `schedule` - specify the desired backup schedule in [crontab format  :octicons-link-external-16:](https://en.wikipedia.org/wiki/Cron)).
+    * `enabled` - set this key to `true`. This enables making the `<backup name>` backup along with the specified schedule.
+    * `storageName` - specify the name of your [already configured storage](backups-storage.md).
+    * `keep` - define the number of backups to keep in the storage. This key is optional.
+    * `type` - specify what [type of backup](backups.md#backup-types) to make. If you leave it empty, the Operator makes a logical backup by default.
+
+    Note that the `percona.com/delete-backup` finalizer and the [` .spec.backup.tasks.[].keep`](operator.md##backuptaskskeep) option are not available for incremental backups.
+
+**Examples**
+
+=== "Logical"    
+
+    This example shows how to set up backups to run every Saturday night and store them in Amazon S3:
+
+    ```yaml
+    ...
+    backup:
+      enabled: true
+      storages:
+        s3-us-west:
+          type: s3
+          s3:
+            bucket: S3-BACKUP-BUCKET-NAME-HERE
+            region: us-west-2
+            credentialsSecret: my-cluster-name-backup-s3
+      tasks:
+       - name: "sat-night-backup"
+         enabled: true
+         schedule: "0 0 * * 6"
+         keep: 3
+         type: logical
+         storageName: s3-us-west
+      ...
+    ```
+
+=== "Physical"    
+    
+    This example shows how to set up backups to run every Saturday night and store them in Amazon S3:
+
+    ```yaml
+    ...
+    backup:
+      enabled: true
+      storages:
+        s3-us-west:
+          type: s3
+          s3:
+            bucket: S3-BACKUP-BUCKET-NAME-HERE
+            region: us-west-2
+            credentialsSecret: my-cluster-name-backup-s3
+      tasks:
+       - name: "sat-night-backup"
+         enabled: true
+         schedule: "0 0 * * 6"
+         keep: 3
+         type: physical
+         storageName: s3-us-west
+      ...
+    ```
+
+=== "Incremental"  
+
+    To run incremental backups, consider the following: 
+
+    1. You must use the same storage for the base backup and subsequent incremental ones
+    2. The `percona.com/delete-backup` finalizer and the [` .spec.backup.tasks.[].keep`](operator.md##backuptaskskeep) option are not available for this backup type.
+
+    This example shows how to set up incremental base backups to run every Sunday at 5 a.m and subsequent incremental backups every night at 1:00 a.m. and store them in Amazon S3:  
+
+    ```yaml
+    ...
+    backup:
+      enabled: true
+      storages:
+        s3-us-west:
+          type: s3
+          s3:
+            bucket: S3-BACKUP-BUCKET-NAME-HERE
+            region: us-west-2
+            credentialsSecret: my-cluster-name-backup-s3
+      tasks:
+       - name: weekly-s3-us-west-incremental
+         enabled: false
+         schedule: "0 1 * * *"
+         keep: 5
```

Comment:
Maybe remove `keep` from example as it is ignored for incremental backups?

Reply:
Done. But it persists for base incremental, right?

Reply:
Yes